packages/utilities/exposure-scanning/patch/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable global-require */
+let octokit;
+
+// Used to limit the number of calls to the patch comment and review comment endpoint
+const maxCommentPatches = 20;
```

Comment:
What prompted this being introduced, did you hit some rate limiting?

Reply:
Not so much rate limiting, but rather I found that the endpoint for updating comments and review comments doesn't take the PR or issue id as an argument (I initially thought it took it as an argument in the body), but instead it gives each comment across the repo a unique id and that's used to identify the comment we'll update. So it's kind of a way of mitigating the scenario you described in the mob review where it goes and updates a load of comments outside the PR/issue in question. It's very unlikely considering the comment id's are obtained from [an endpoint](https://docs.github.com/en/rest/reference/pulls#list-review-comments-on-a-pull-request) which **does** take in the PR/issue id, but I think it's useful for if someone were to change the way that the fetch and patch systems interact

Reply:
Thanks for the explaination @HarryVerhoef you might want to add why you want to limit the calls in the comment as you've explained here